bin/node/src/commands/bundled.rs

```diff
@@ -62,7 +62,7 @@ pub enum BundledCommand {
         block_producer: BlockProducerConfig,
 
         #[command(flatten)]
-        ntx_builder: NtxBuilderConfig,
+        ntx_builder_config: NtxBuilderConfig,
```

Comment:
I'm not sure we need to add the `_config` suffix here. For example, for the `BlockProducerConfig` above, we name the field just `block_producer`. I think we can probably keep the `ntx_builder` name here.